<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestItem</class>
 <widget class="QWidget" name="RequestItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>86</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	background-color: #595b5d;
	color: #bbbbbb;
	border: none;
	padding: 5px;
	border-radius: 8px;
}

#RequestItem {
	background: none;
	background-color: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QLabel" name="verbLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>GET</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="urlLabel">
       <property name="text">
        <string>URL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="uriLabel">
     <property name="styleSheet">
      <string notr="true">#uriLabel {
	background-color: #445F68;
}</string>
     </property>
     <property name="text">
      <string>URI</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
